### PR TITLE
lodash-3.10: Add _.reduce for LoDashExplicitObjectWrapper

### PR DESCRIPTION
**Pull request by zuzusik**

### CODE REVIEW

File: lodash/lodash-3.10.d.ts [hunk 1]
```diff
@@ -7732,6 +7732,53 @@ declare module _ {
...
     }
 
+interface LoDashExplicitObjectWrapper<T> {
+         /**
+        * @see _.reduce
+        **/
+        reduce<TValue, TResult>(
+            callback: MemoIterator<TValue, TResult>,
+            accumulator: TResult,
+            thisArg?: any): TResult;
```
Comment (zuzusik) [inline lodash/lodash-3.10.d.ts]: err, this should be `LoDashExplicitArrayWrapper<TResult>`, will update now

**Comment by vvakame** [comment]
We can't get response from authors.
please ping me again in 2 days. I'll merge this PR.

**Comment by zuzusik** [comment]
@vvakame ping

**Comment by zuzusik** [comment]
@vvakame should we merge this?

**Pull request merged**